src/features/ClinicDashboard/components/Cards/AppointmentsCard.js

```diff
@@ -27,24 +29,27 @@ const AppointmentUserArea = styled(CardUserArea)`
 
 const AppointmentsCard = function ({
   data: {
-    avatar, name, specialist, time, symptoms,
+    doctor: {
+      photo, first_name, last_name, specialization_name,
+    }, visit_date, reason,
   },
 }) {
   const [isShow, setShow] = useState(false);
```

Comment:
называй переменные чуть более подробно. и show в данном случае больше как глагол, а как определение visible будет более подходящим, при том что потом ты как раз передаешь это в параметр `visible`
```suggestion
  const [isSettingsVisible, setIsSettingsVisible] = useState(false);
```

Reply:
+